frontend/src/types/index.ts

```diff
@@ -103,4 +103,9 @@ export type Ride = {
   endTime: string;
   rider: Rider;
   driver?: Driver;
+  recurring: boolean;
+  recurringDays?: number[];
+  endDate?: string;
+  deleted?: boolean;
```

Comment:
oops i think the API docs might not be up to date, we changed `deleted` to be an array of string dates